python/meep.i

```diff
@@ -1346,29 +1366,19 @@ void _get_eigenmode(meep::fields *f, double omega_src, meep::direction d, const
 // Make omega members of meep::dft_ldos available as 'freq' in python
 %extend meep::dft_ldos {
 
-    double get_omega_min() {
-        return $self->omega_min;
-    }
-    double get_domega() {
-        return $self->domega;
-    }
     int get_Nomega() {
         return $self->Nomega;
     }
     %pythoncode %{
         def freqs(self):
             import math
             import numpy as np
-            start = self.omega_min / (2 * math.pi)
-            stop = start + (self.domega / (2 * math.pi)) * self.Nomega
-            return np.linspace(start, stop, num=self.Nomega, endpoint=False).tolist()
+            start = self.omega[0] / (2 * math.pi)
+            stop = self.omega[-1] / (2 * math.pi)
+            return np.linspace(start, stop, num=self.Nomega, endpoint=True).tolist()
```

Comment:
After the change to `std::vector`, just `return (self.omega / (2*math.pi)).tolist()`